video-presentation.py

```diff
@@ -246,13 +286,19 @@ def find_slides_transitions(self, connector, host, secret_key, resource, masks=N
             self.logger.debug("Using advanced algorithm for finding slides. settings: %s", settings)
             results = self.slide_find_advanced(resource['local_paths'][0], masks=masks, **settings)
 
+        # Create and upload the compressed previews
+        mp4_preview, webm_preview = self.create_video_previews(resource['local_paths'][0])
```

Comment:
if you want to speed things up, this can be done in parallel with the slide detection?

Reply:
This would indeed make a big difference but I wouldn't be confident that I can do that well (I ahven't done threading in python before).

Reply:
Implemented in #3